scio-core/src/main/java/com/spotify/scio/transforms/FutureHandlers.java

```diff
@@ -44,7 +44,6 @@ public interface Guava<V> extends Base<ListenableFuture<V>, V> {
     @Override
     default void waitForFutures(Iterable<ListenableFuture<V>> futures)
         throws InterruptedException, ExecutionException {
-      // Futures#allAsList only works if all futures succeed
```

Comment:
i think this comment was on an old code that used `Futures#allAsList` instead of the current `Futures.successfulAsList`. As the comment mentions `allAsList` fails if any future fails while `successfulAsList` returns a null value for any failed future.

Reply:
This is not old if you check git blame. This comment was just a hit why we do NOT use `Futures#allAsList`. maybe should have been turned into
```
use Future#successfulAsList instead of Futures#allAsList which only works if all futures succeed
```